internal/ngrokapi/edge_modules_https.go

```diff
@@ -50,13 +57,58 @@ func (c *defaultHTTPSEdgeModulesClientset) TLSTermination() *https_edge_tls_term
 	return c.tlsTermination
 }
 
+type RawHTTPSModuleSetPolicy json.RawMessage
```

Comment:
Let's import this from `api/ngrok/v1alpha1/common.go`, since we'll eventually change this type it would be nice if we only had to change it in one spot.

Reply:
This was a client alias that was helpful for `ngrokapi`, and I decided to drop it to avoid confusion and simplify